
{'date': 'Wed Oct 09 2019 16:59:34.941 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from soraya.main import Bloco
  module soraya.main line 39
    Mapa1()
  module soraya.main line 34
    loc.entra(MAPA0)
  module _spy.vitollino.main line 471
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'str' object has no attribute 'nome'
'''},
{'date': 'Wed Oct 09 2019 16:59:41.287 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from soraya.main import Bloco
ImportError: cannot import name 'Bloco'
'''},
{'date': 'Wed Oct 09 2019 17:00:31.447 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 5
    from soraya.main import Bloco
  module soraya.main line 39
    Mapa1()
  module soraya.main line 34
    loc.entra(MAPA0)
  module _spy.vitollino.main line 471
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'str' object has no attribute 'nome'
'''},
{'date': 'Sun Aug 30 2020 16:01:45.608 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from soraya.main import Bloco
  module soraya.main line 39
    Mapa1()
  module soraya.main line 34
    loc.entra(MAPA0)
  module _spy.vitollino.main line 482
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'str' object has no attribute 'nome'
'''},
{'date': 'Sun Aug 30 2020 16:02:03.303 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 177
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from soraya.main import Bloco
ImportError: cannot import name 'Bloco'
'''},
{'date': 'Thu Jul 14 2022 09:30:30.369 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 4
    from soraya.main import Bloco
  module soraya.main line 39
    Mapa1()
  module soraya.main line 34
    loc.entra(MAPA0)
  module _spy.vitollino.main line 482
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'str' object has no attribute 'nome'
'''},